src/pathways/pathway/pathways.service.ts

```
@@ -440,7 +485,80 @@ export class PathwaysService {
     response: Response
   ): Promise<Response> {
     const apiId = APIID.PATHWAY_LIST;
+    const pathwayListCacheTtl = parseInt(
+      this.configService.get('PATHWAY_LIST_CACHE_TTL_SECONDS') || '1800',
+      10
+    );
     try {
+      const cacheKey = this.generatePathwayListCacheKey(tenantId, organisationId, listPathwayDto);
```

ESLint

[error] Replace `tenantId,·organisationId,·listPathwayDto` with `⏎········tenantId,⏎········organisationId,⏎········listPathwayDto⏎······`

(prettier/prettier)